api.planx.uk/saveAndReturn/validateSession.test.ts

```diff
@@ -155,16 +156,410 @@ describe("Validate Session endpoint", () => {
         expect(response.body).toHaveProperty("removedBreadcrumbs", {});
       });
   });
+
+  // TODO this may be an existing issue or it may be my misunderstanding of how this should work
+  it.skip("returns changed nodes and invalidated breadcrumbs when a flow has updated only nested nodes (answers but not questions)", async () => {
```

Comment:
Did I get this test right? Is this a pre-existing bug?

Reply:
Your test looks correct to me - this looks like a pre-existing bug.

If editors update the text an of an answer (e.g. "Yes" to "No"), then we would require users to re-visit this question and provide a new answer.

Reply:
This is a good catch! It seems reconciliation is only properly catching cases where the responses have changed order or changed in number (because that bubbles up as a diff in the `edges` of the parent question), but not the case where an existing response attribute changes alone (`alteredNodes` is properly populated in the validate-session response, but `removedBreadcrumbs` is not)

Addressing in separate PR makes sense to me, thanks for stubbing out this failing test :+1: